tests/git/test_git_commands.py

```diff
@@ -587,20 +589,29 @@ def test_execute(
 ):
     mock_execute.return_value = mock_cursor(
         [
-            {"name": "repo/branches/main/a/S3.sql"},
-            {"name": "repo/branches/main/s1.sql"},
-            {"name": "repo/branches/main/s2"},
+            {"name": "/a/S3.sql"},
+            {"name": "/s1.sql"},
+            {"name": "/s2"},
         ],
         [],
     )
 
     result = runner.invoke(["git", "execute", repository_path])
 
     assert result.exit_code == 0, result.output
-    ls_call, *execute_calls = mock_execute.mock_calls
-    assert ls_call == mock.call(f"ls {expected_stage}", cursor_class=DictCursor)
+    create_call, copy_call, ls_call, *execute_calls = mock_execute.mock_calls
+    assert create_call == mock.call(
+        "create temporary stage if not exists IDENTIFIER('FOO.BAR.snowflake_cli_tmp_stage_123')"
+    )
+    assert copy_call == mock.call(
+        f"copy files into @FOO.BAR.snowflake_cli_tmp_stage_123/ from {expected_stage}"
+    )
```

Comment:
Why you copy sql files to temporary stage? They don't need it.

Reply:
I'm copying git contents to stage. I don't differentiate if this is SQL or Python.